r2rap/python/remote.py

```diff
@@ -140,21 +144,45 @@ def _handle_client(self, c):
 					break
 				if len(buf) == 0:
 					print "Connection closed\n"
-					break
+					return -1
 				self._handle_packet(c, ord(buf))
 			except KeyboardInterrupt:
 				break
 
+	def listen_stdin(self):
+		line = sys.stdin.readline()
+
+		def threaded_client(line):
+			cl = RapClient("localhost", 9999)
+			cl.cmd(line)
+
+		threading.Thread(target=threaded_client, args=(line.strip(), )).start()
```

Comment:
i would prefer not to create threads from this library, this can be delegated to the user who uses the api.